pkgs/servers/web-apps/plausible/default.nix

```diff
@@ -5,43 +5,35 @@
 , glibcLocales
 , cacert
 , mkYarnModules
+, fetchYarnDeps
 , nodejs
-, fetchpatch
 , nixosTests
 }:
 
 let
   pname = "plausible";
-  version = "1.3.0";
+  version = "1.4.0";
 
   src = fetchFromGitHub {
     owner = "plausible";
     repo = "analytics";
     rev = "v${version}";
-    sha256 = "03lm1f29gwwixnhgjish5bhi3m73qyp71ns2sczdnwnbhrw61zps";
+    sha256 = "sha256-mMQqjl3rNwjQGx8SN9PAkQErwiCs1HYSP4HWzevxYbQ=";
   };
 
   # TODO consider using `mix2nix` as soon as it supports git dependencies.
   mixFodDeps = beamPackages.fetchMixDeps {
     pname = "${pname}-deps";
     inherit src version;
-    sha256 = "1x0if0ifk272vcqjlgf097pxsw13bhwy8vs0b89l0bssx1bzygsi";
-
-    # We need ecto 3.6 as this version checks whether the database exists before
-    # trying to create it. The creation attempt would always require super-user privileges
-    # and since 3.6 this isn't the case anymore.
-    patches = [
-      ./ecto_sql-fix.patch
-      ./plausible-Bump-clickhouse_ecto-dependency-to-be-compatible-with-ecto-3.6.patch
-    ];
+    sha256 = "sha256-ZQfrTxsLzCWFf3vabOk0vyHWZLw69GJovm3vR+7UbMY=";
   };
```

Comment:
Does this still work fine with the update script?

Reply:
Just updated the script to make sure it works with the new version.